### DISCUSSION TITLE
Get font information of text in image

### DISCUSSION THREAD

**Discussion by ashokparmar29** [PaddlePaddle/PaddleOCR#13379] [Q&A] [answered]
Hi everyone,
I'm interested in using PaddleOCR for a project that involves extracting text from images. However, in addition to the detecting text, I'd also like to identify the font information (e.g., font family, style) if possible.
Is there a way to leverage PaddleOCR or any additional libraries/techniques to achieve font identification alongside text recognition within the workflow?
Any insights or suggestions would be greatly appreciated!

**Comment by UserWangZz** [comment, marked as answer]
Regarding your font recognition, style, etc., you need to design a classifier for classification. The recognition model is only used to recognize text and does not have the ability to recognize fonts, styles, etc.

**Reply by 99bits** [reply]
Thank you @UserWangZz for your answer.  Appreciate it. 
Can you give more details of what the dataset for training the font classifcation would look like and any other details that you can think of that could help.  

Cheers

**Reply by UserWangZz** [reply]
Select an appropriate backbone and corresponding font label data. This is a simple classification task.

**Comment by ashokparmar29** [comment]
Hi @UserWangZz 
Thanks for the suggestion. 
I came across https://scribeocr.com/. The ScribeOCR  includes functionality that seems capable of extracting and replicating font information, suggesting it can identify and apply font styles during text recognition.
can we implement or integrate the same with PaddleOCR?
github repo: https://github.com/scribeocr/scribeocr

**Reply by UserWangZz** [reply]
Font style classification is a simple classification task. At the same time, there are hundreds or thousands of fonts style in the same language, and some fonts are even very similar. To implement a general font classification model requires a large amount of data and labels, which is expensive. You can simply implement a font classifier for your own task based on your scenario.

If you are interested in integrating scribeocr into paddleocr, we will provide you with certain technical support. Thank you for your contribution to the paddleocr community.

**Reply by ashokparmar29** [reply]
Thank you for the suggestion. I am trying to understand how scribeocr do this and then I will try to implement same in the PaddleOCR. I will ask for help after understanding the scribeocr.

**Reply by UserWangZz** [reply]
Thank you for your contribution to the paddleocr community.
